app/src/main/java/net/gsantner/markor/format/general/CommonTextActions.java

```diff
@@ -55,9 +61,16 @@
     private final Activity _activity;
     private final HighlightingEditor _hlEditor;
 
+    protected AppSettings _appSettings;
+    private int _tabWidth;
+    protected Context _context;
+
     public CommonTextActions(Activity activity, HighlightingEditor hlEditor) {
         _activity = activity;
         _hlEditor = hlEditor;
+        _context = activity != null ? activity : _hlEditor.getContext();
```

Comment:
I cant find somethinz requiring saving context as member. This can be removed right? Notice: It may leak memory/data